specification/dimensions/servicename.md

```diff
@@ -1,10 +1,10 @@
 # Service Name
 
-Service Name is an identifier assigned to a service by the provider. The Service Name is commonly used for cost
-reporting, analysis and allocation scenarios.
+A service represents the offering that was purchased through the provider (e.g., professional or cloud services). For cloud resources, the service is a higher-level classification of the resource type and may include multiple usage charges of different types (e.g., compute, storage, networking). The service is not a classification of the type of usage. Instead, it represents the offering you might find on a provider's website.
```

Comment:
```suggestion
A service represents an offering that encapsulates resources and/or services that can be purchased from a provider (e.g., cloud virtual machine offering, a SaaS database offering or professional services offering from a systems integrator). With in a service, providers can charge for usage of various resources, capabilities and features consumed (e.g. a cloud database service may provide multiple charges for compute, storage and networking consumed by a resource).
```

Reply:
I feel like "encapsulates resources and/or services" adds an unnecessary layer of ambiguity here 🤔

Reply:
Let me know what you think of this...